swan-cern/files/ca-certs/ca-bundle.crt

```diff
@@ -0,0 +1,96 @@
+# CERN Root Certification Authority 2
+-----BEGIN CERTIFICATE-----
```

Comment:
This expires in 2033 ?

Reply:
Yes, the expiration time is: Mar 19 13:05:34 2033 GMT